validation/vertical_mixing_closures/heterogeneous_windy_convection.jl

```diff
@@ -0,0 +1,157 @@
+using Printf
+using GLMakie
+using Oceananigans
+using Oceananigans.Units
+using Oceananigans.TurbulenceClosures: CATKEVerticalDiffusivity
+using Oceananigans.TurbulenceClosures: ExplicitTimeDiscretization
+
+Nx = 1
+Ny = 64
+Nz = 32
+
+const Lx = 100kilometers
+const Ly = Lx
+const Lz = 256
+
+grid = RectilinearGrid(size = (Nx, Ny, Nz),
+                       x = (0, Lx),
+                       y = (0, Ly),
+                       z = (-Lz, 0),
+                       topology=(Periodic, Bounded, Bounded))
+
+@show grid
+@inline Qᵇ(x, y, t) = 2e-8 #* sin(2π * y / Ly)
+@inline Qᵘ(x, y, t) = -1e-4 * sin(π * y / Ly)
+
+b_top_bc = FluxBoundaryCondition(Qᵇ)
+u_top_bc = FluxBoundaryCondition(Qᵘ)
+
+b_bcs = FieldBoundaryConditions(top=b_top_bc)
+u_bcs = FieldBoundaryConditions(top=u_top_bc)
+
+etd = ExplicitTimeDiscretization()
+closure = CATKEVerticalDiffusivity()
+
+model = HydrostaticFreeSurfaceModel(; grid, closure,
+                                    #coriolis = FPlane(f=1e-4),
+                                    momentum_advection = WENO(),
+                                    #tracer_advection = (b=WENO(), e=CenteredSecondOrder()),
+                                    tracer_advection = (b=WENO(), e=WENO()),
+                                    #tracer_advection = (b=WENO(), e=nothing),
```

Comment:
what's with the commented ones here?

Reply:
Hmm those were for testing I'll get rid of them